examples/cpp/TCPSendStack.cc

```diff
@@ -27,7 +27,7 @@ struct stack_key_t {
   int kernel_stack;
 };
 
-BPF_STACK_TRACE(stack_traces, 10240)
+BPF_STACK_TRACE(stack_traces, 10240);
```

Comment:
The `BPF_STACK_TRACE` defines like:
```
#define BPF_STACK_TRACE(_name, _max_entries) \
  BPF_TABLE("stacktrace", int, struct bpf_stacktrace, _name, _max_entries);
```
There is already a `;` at the end.

I do recognize the usage is inconsistent (e.g., at tools directory below)
```
-bash-4.2$ egrep -r BPF_STACK_TRACE
deadlock_detector.c:BPF_STACK_TRACE(stack_traces, 655360);
memleak.py:BPF_STACK_TRACE(stack_traces, 10240)
offcputime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
offwaketime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
old/profile.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
profile.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
stacksnoop.lua:BPF_STACK_TRACE(stack_traces, 128)
trace.py:                stack_table = "BPF_STACK_TRACE(%s, 1024);" % self.stacks_name \
wakeuptime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
stackcount.py:BPF_STACK_TRACE(stack_traces, 1024);
```

Ideally, we may want to change macro instead to force user to add `;` just like `BPF_TABLE`.  This risks breaking some existing codes. Any other concerns?

Reply:
I see. I "fixed" the macro in my local branch. This does make things tricky. Maybe we just keep this part as is.